apps/tome-api/src/features/card/card.fns.ts

```diff
@@ -750,6 +775,173 @@ export const deck: DbCard[] = [
 			},
 		},
 	},
+	{
+		id: '55',
+		name: 'Burning flame',
+		type: 'spell',
+		colors: ['red'],
+		attack: 18,
+		description:
+			'Every time this spell deals damage, discard the bottom card from its stack. If there’s no more cards to discard, discard Burning flame.',
+		effects: {
+			onDealDamage: async function* ({ actions, game, ownerSide, thisCard }) {
+				const stack = STACKS.find(stack => game.board.players[ownerSide].stacks[stack].includes(thisCard));
+				if (!stack) {
+					yield* actions.discard({ card: thisCard, from: game.board.field, side: ownerSide });
+					return;
+				}
+				const cardToDiscard = game.board.players[ownerSide].stacks[stack].pop();
+				if (cardToDiscard) {
+					yield* actions.discard({
+						card: cardToDiscard,
+						from: game.board.players[ownerSide].stacks[stack],
+						side: ownerSide,
+					});
+				}
+			}
+		},
+	},
+	{
+		id: '57',
+		name: 'Wind Sceptre',
+		type: 'spell',
+		colors: ['blue', 'green'],
+		attack: 30,
+		description: 'When this card is revealed, draw one card. Then discard this card.',
+		effects: {
+			onReveal: async function* ({ actions, ownerSide }) {
+				yield* actions.draw({ sides: [ownerSide] });
+			},
+			beforeCombat: async function* ({ actions, game, thisCard }) {
+				yield* actions.discard({ card: thisCard, from: game.board.field, side: 'sideA' });
```

Comment:
this isn't a field card, this will fail! try just discarding it onReveal as well, and remove it from the turn `casts` instead